lib/archethic/bootstrap.ex

```diff
@@ -213,6 +213,7 @@ defmodule Archethic.Bootstrap do
     SelfRepair.start_scheduler()
 
     :persistent_term.put(:archethic_up, :up)
+    Archethic.PubSub.notify_node_up(:persistent_term.get(:archethic_up, nil))
```

Comment:
Why are you passing parameter to this function ?
If you are persistent_term.put(:archethic_up, :up), it will be everytime up.
So you should just send message :node_up